cypress/integration/docMap.spec.ts

```diff
@@ -0,0 +1,78 @@
+describe('Editor docs tests', () => {
```

Comment:
It's better to have behaviour tests for the following scenarios than hard coding every possible test here (Avoid Duplication). 

1. Default loading
2. Test to see if the correct URL loads when changing from one diagram to other (One or two only).
3. Test to check URLs for a case where config URL doesn't exist.